src/luascript.cpp

```diff
@@ -9391,56 +9391,46 @@ int LuaScriptInterface::luaPlayerHasMount(lua_State* L) {
 	return 1;
 }
 
-int LuaScriptInterface::luaPlayerGetPremiumDays(lua_State* L)
+int LuaScriptInterface::luaPlayerGetPremiumTime(lua_State* L)
 {
-	// player:getPremiumDays()
+	// player:getPremiumTime()
 	Player* player = getUserdata<Player>(L, 1);
 	if (player) {
-		lua_pushnumber(L, player->premiumDays);
+		lua_pushnumber(L, std::max<int32_t>(0, player->premiumEndsAt - time(nullptr)));
```

Comment:
```suggestion
        #include <climits>
		lua_pushnumber(L, std::max<int32_t>(INT_MAX, player->premiumEndsAt - time(nullptr)));
```
Maybe cap at max int32 instead of resetting to zero?

Reply:
isn't that equivalent to std::numeric_limits<int32_t>::max()?